src/common/darktable.c

```diff
@@ -144,9 +144,9 @@ static int usage(const char *argv0)
 #ifdef HAVE_OPENCL
   printf("  --disable-opencl\n");
 #endif
-  printf("  --dump-pfm <module>\n");
-  printf("  --dump-pipe <module>\n");
-  printf("  --bench-module <module>\n");
+  printf("  --dump-pfm <module,moduleb>\n");
```

Comment:
just to be picky I'll use `<modulea,moduleb>`

Reply:
right :-) I was thinking if there is some common form for this ?